src/main/java/net/clgd/ccemux/plugins/builtin/CCEmuXAPI.java

```diff
@@ -50,7 +54,33 @@ public API(CCEmuX emu, EmulatedComputer computer, String name) {
 
 			methods.put("openEmu", o -> {
 				int id = ArgumentHelper.optInt(o, 0, -1);
-				EmulatedComputer ec = emu.createComputer(b -> b.id(id));
+				String program = ArgumentHelper.optString(o, 1, null);
+				
+				EmulatedComputer ec = emu.createComputer(b -> b.id(id), program == null);
+				
+				if (program != null) {
+					IWritableMount mount = ec.getRootMount();
+					
+					try {
+						if (!mount.exists(program)) {
+							emu.removeComputer(ec);
+							throw new LuaException("program not found");
+						}
+						
+						if (!mount.isDirectory("startup/")) mount.makeDirectory("startup");
+						
+						try (InputStream src = mount.openForRead(program);
+							 OutputStream dst = mount.openForWrite("startup/0-ccemux" + ".lua")) {
+							dst.write("fs.delete(\"startup/0-ccemux.lua\")".getBytes());
```

Comment:
Being _really_ fussy now, but can you do `.getBytes(StandardCharsets.UTF_8)`? I believe windows defaults to something else so it'd be good to be consistent. Also add a trailing `;` after the `fs.delete` call so we don't break on programs starting with `(function()`.